website/docs/patterns/disabled-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,104 @@
+## Permissions 
+
+When a user does not have permissions, hide the related actions, navigation items, and views.
+
+### Hide/show relevant UI
+
+In this example, the user's role is **viewer**, and they cannot deploy or destroy a cluster. An explanation is unnecessary as users do not require additional functionality for their day-to-day tasks.
+
+![Image showing two open dropdown lists with number of items shown based on user permissions, one with only one menu item, while the other has more options that allow for further capabilities.](/assets/patterns/disabled-patterns/permissions-example.png)
+
+Similarly, if a user has the permissions to update, show the relevant UI allowing them to do just that.
+
+
+### Provide guidance
+
+When a user lands on a page they can't access, indicate why it's unavailable and provide guidance so the user can fix the error. Consider displaying a call to action so the user doesn't feel stuck.
+
+![Showing a 403 error page with information about what is going on, with a way to go back and a way to log in with a different token](/assets/patterns/disabled-patterns/permissions-example-3.png)
+
+## Conditional availability
+
+When other steps must be taken to enable a feature or function, provide context in place of that feature so users can enable it.
+
+!!! Dont
+
+In this example, the button is disabled without context, leaving the user confused and unaware of the steps to enable the feature.
+
+![Showing an interface with a disabled button that doesn't describe why that button is disabled. This is not a recommended experience.](/assets/patterns/disabled-patterns/conditional-availability-example-disabled.png)
+
+Disabled elements lack interactive states, preventing users from hovering, focusing, or clicking. For users with assistive technology, e.g., a screen reader, disabled elements (including their text) are almost entirely ignored.
+
+!!!
+
+
+!!! Do
+
+Instead, we recommended replacing the disabled element with contextual guidance to help users understand why this feature isn't available, and providing them with clear steps to enable it.
+
+![Showing an interface with a compact alert explaining why the expected behavior isn't there with guidance.](/assets/patterns/disabled-patterns/conditional-availability-example-explanation.png)
+
+If possible, provide CTAs directly to the place where users can take immediate action.
+
+!!!
+
+
+## Upgrading
+
+When upgradable features are available and a user has the necessary permissions, highlighting a path to upgrade provides a graceful marketing opportunity. However, be mindful not to overdo it and make the user feel overly advertised to.
+
+There are two ways to showcase an upgradable feature:
+
+- Space to explain
```

Comment:
I still think "Space to explain" is somewhat ambiguous and unclear.

How about something more along the lines of:
Feature highlight, Feature explanation, Feature spotlight
or similar?

Reply:
Btw, the update to the related image makes it _much_ clearer. Thanks!

Reply:
I like "feature spotlight" ill update!